tools/developer.html

```diff
@@ -108,37 +219,56 @@ <h3>Markup</h3>
         editor.find('.rendering_stats').text(rendering_stats);
         editor.find('.rendering_time').text(rendering_time);
         editor.find('output').text(result.value);
-        SourceStore.save(source, language);
+        saveSettings();
       });
 
       $('.editor button#show-structure').click(function(e) {
         var editor = $(this).parents('.editor');
         editor.toggleClass('visible-structure');
+
+        localStorage.setItem(key_STRUCTURE,
+          editor.hasClass('visible-structure'));
       });
     });
 
     var SourceStore;
+    var key_CODE = 'hljs-source';
+    var key_LANGUAGE = 'hljs-lang';
+    var key_THEME = 'hljs-theme';
+    var key_STRUCTURE = 'hljs-structure';
     (function(){
       SourceStore = {
-        save: function(source, lang){
-          localStorage.setItem(key_SOURCE, source);
-          localStorage.setItem(key_LANG, lang);
+        save: function({code, language, theme}){
+          localStorage.setItem(key_CODE, code);
+          localStorage.setItem(key_LANGUAGE, language);
+          localStorage.setItem(key_THEME, theme);
         },
         resolve: function(){
-          return [
-            localStorage.getItem(key_SOURCE),
-            localStorage.getItem(key_LANG)
-          ];
+          let code = localStorage.getItem(key_CODE)
+          if (!code) return null;
+
+          return {
+            code,
+            language: localStorage.getItem(key_LANGUAGE),
+            theme: localStorage.getItem(key_THEME),
```

Comment:
```suggestion
            theme: localStorage.getItem(key_THEME) || 'default.css',
```

The page loads without a default style if we rely solely on local storage.